<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add To Timeline</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../images/openshot.qrc">
    <normaloff>:/openshot.svg</normaloff>:/openshot.svg</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="vbox_parent">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="vboxTreeParent">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QPushButton" name="btnMoveUp">
               <property name="toolTip">
                <string>Move Up</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset theme="go-up">
                 <normaloff/>
                </iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnMoveDown">
               <property name="toolTip">
                <string>Move Down</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset theme="go-down">
                 <normaloff/>
                </iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnShuffle">
               <property name="toolTip">
                <string>Shuffle</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset theme="view-refresh">
                 <normaloff/>
                </iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnRemove">
               <property name="toolTip">
                <string>Remove</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset theme="list-remove">
                 <normaloff/>
                </iconset>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="vbox_properties">
           <item>
            <widget class="QLabel" name="lblTimelineLocation">
             <property name="styleSheet">
              <string notr="true">font-weight: bold</string>
             </property>
             <property name="text">
              <string>Timeline Location</string>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="lblStartTime">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>Start Time (seconds):</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="txtStartTime">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>150</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximum">
                <double>999999999.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="lblTrack">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>Track:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cmbTrack">
               <property name="minimumSize">
                <size>
                 <width>150</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <property name="topMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>Image Length (seconds):</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="txtImageLength">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>150</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximum">
                <double>999999999.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="lblFade">
             <property name="styleSheet">
              <string notr="true">font-weight: bold; margin-top: 15px;</string>
             </property>
             <property name="text">
              <string>Fade</string>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="lblFade1">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>Fade:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cmbFade">
               <property name="minimumSize">
                <size>
                 <width>150</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLabel" name="lblFadeLength">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>Length (seconds):</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="txtFadeLength">
               <property name="minimumSize">
                <size>
                 <width>150</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximum">
                <double>999999999.000000000000000</double>
               </property>
               <property name="value">
                <double>2.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="styleSheet">
              <string notr="true">font-weight: bold; margin-top: 15px;</string>
             </property>
             <property name="text">
              <string>Zoom</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <property name="topMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Zoom:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cmbZoom">
               <property name="minimumSize">
                <size>
                 <width>150</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="lblTransition">
             <property name="styleSheet">
              <string notr="true">font-weight: bold; margin-top: 15px;</string>
             </property>
             <property name="text">
              <string>Transition</string>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLabel" name="lblTransition1">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>Transition:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cmbTransition">
               <property name="minimumSize">
                <size>
                 <width>150</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>150</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QLabel" name="lblTransitionLength">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>Length:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="txtTransitionLength">
               <property name="minimumSize">
                <size>
                 <width>150</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximum">
                <double>999999999.000000000000000</double>
               </property>
               <property name="value">
                <double>2.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="lblTotalLength">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Total Length (seconds):</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblTotalLengthValue">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font-weight:bold;</string>
         </property>
         <property name="text">
          <string>0.0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="btnBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
       <property name="centerButtons">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../images/openshot.qrc"/>
 </resources>
 <connections/>
</ui>
